ecs-cli/modules/cli/compose/context/context.go

```diff
@@ -40,7 +40,8 @@ type Context struct {
 	EC2Client ec2client.EC2Client
 
 	// IsService would decide if the resource created by this compose project would be ECS Tasks directly or through ECS Services
-	IsService bool
+	IsService   bool
+	TaskRoleArn string
```

Comment:
@yinshiua I agree, however I wasn't sure where to put it since it can't go in `libcompose`'s `Project`. Do you have any suggestions on where it should go?